lib/iris/_merge.py

```diff
@@ -1466,9 +1473,11 @@ def _get_cube(self, data):
                                for coord, dims in self._aux_coords_and_dims]
         kwargs = dict(zip(iris.cube.CubeMetadata._fields, signature.defn))
 
+        cmds = self._cell_measures_and_dims
```

Comment:
@marqh Careful ... you need to copy the cell measures, otherwise multiple **different** (merged) cubes will end up sharing the **same** cell methods :bomb: